packages/web-pkg/src/components/CreateLinkModal.vue

```diff
@@ -191,16 +136,15 @@ export default defineComponent({
     const { $gettext } = language
     const passwordPolicyService = usePasswordPolicyService()
     const { isEnabled: isEmbedEnabled, postMessage } = useEmbedMode()
-    const { expirationRules } = useExpirationRules()
     const {
       defaultLinkType,
       getAvailableLinkTypes,
       getLinkRoleByType,
       isPasswordEnforcedForLinkType
     } = useLinkTypes()
     const { addLink } = useSharesStore()
-    const contextMenuDrop = ref<InstanceType<typeof OcDrop>>()
     const advancedMode = ref(false)
+    const invalidExpiryDate = ref(false)
```

Comment:
I'd like to have

```suggestion
    const isInvalidExpiryDate = ref(false)
```

here too, but on the other hand, `advancedMode` also doesn't follow that naming scheme. 😞